presto-main/src/main/java/io/prestosql/security/ViewAccessControl.java

```diff
@@ -74,4 +78,14 @@ public void checkCanGrantExecuteFunctionPrivilege(SecurityContext context, Strin
     {
         return delegate.getColumnMasks(context, tableName, columnName, type);
     }
+
+    private void executeAsViewOwner(Consumer<AccessControl> consumer)
+    {
+        try {
+            consumer.accept(delegate);
+        }
+        catch (AccessDeniedException e) {
+            throw new PrestoException(PERMISSION_DENIED, "Access Denied: View owner does not have sufficient privileges", e);
```

Comment:
This:

```suggestion
            throw new PrestoException(PERMISSION_DENIED, "Access Denied: View owner does not have sufficient privileges: " + e.getMessage(), e);
```

would be more useful & more actionable.

Reply:
@findepi Could you please note in what case it will be more useful and what additional value it brings?
I think it is not and as the same information is stored in the rethrown exception e and stores in stacktrace.

Reply:
updated.